src/network.zig

```diff
@@ -972,20 +972,38 @@ pub const Protocols = struct {
 					game.Player.setPosBlocking(try reader.readVec(Vec3d));
 				},
 				.worldEditPos => {
-					const typ = try reader.readEnum(WorldEditPosition);
-					switch(typ) {
-						.selectedPos1, .selectedPos2 => {
-							const pos = try reader.readVec(Vec3i);
-							switch(typ) {
-								.selectedPos1 => game.Player.selectionPosition1 = pos,
-								.selectedPos2 => game.Player.selectionPosition2 = pos,
-								else => unreachable,
-							}
-						},
-						.clear => {
-							game.Player.selectionPosition1 = null;
-							game.Player.selectionPosition2 = null;
-						},
+					if(isServerSide(conn)) {
```

Comment:
SHould be a member function
```suggestion
					if(conn.isServerSide()) {
```
Also since you added this helper function, please in the future change it in all places that currently use `conn.user != null` for this purpose.